### ISSUE TITLE
Wrong chapters order

### ISSUE THREAD

**Issue by szaffarano** [bmc/ebook-template#1]
The build script does not order the chapters correctly.  I fixed this issue sorting the glob output in `build` script.  I'm attaching a patch file with this change:

```diff
-CHAPTERS                   = glob(os.path.join(BOOK_SRC_DIR, 'chapter-*.md'))
+CHAPTERS                   = sorted(glob(os.path.join(BOOK_SRC_DIR, 'chapter-*.md')))
```

**Comment by bmc** [comment]
Great find. Thanks. Patching now.

**Comment by bmc** [comment]
Fixed in v0.6.1
